portfolio/src/main/webapp/CommentHandler.js

```diff
@@ -0,0 +1,31 @@
+export default class CommentHandler {
+  /**
+   * Loads all comments from backend.
+   * This function is called when the page is loaded.
+   * @param {HTML div object} commentBox The div in document with id "comment-box"
+   * @param {HTML template} commentCell The template for one comment
+   * @private
+   */
+  async loadComments_(commentBox, commentCell) {
+    let commentResponse = undefined;
+    try {
+      commentResponse = await fetch('/comment');
+    } catch (e) {
+      throw e;
+      return;
+    }
+
+    try {
+      const commentArray = await commentResponse.json();
+      let comments = commentArray
+        .map(entry => {
+          let clone = commentCell.content.cloneNode(true);
```

Comment:
This won't return anything. Multiline anonymous arrow functions need explicit return lines.

Reply:
yeah that variable is not used. I forgot to delete it :)

Reply:
In case that the variable is not used, Arrays.map doesn't seem like the most logical function here, since you don't really want the resulting returned array, which will cost memory. Use Arrays.Foreach instead.

Reply:
Oh thanks! Just changed it.